### ISSUE TITLE
New Homepage and Checklist

### ISSUE THREAD

**Issue by c-schwarze** [betheluniversity/portal#50]
@johjacb and I will work on getting this started. To quick summarize:

* Add notification button to the top right
* Change the dashboard to be a mini dashboard with a button
* Both the notification and the dashboard take us to a separate page (tab?) with the checklist

Jake, let me know how you want to go about getting this started.

**Comment by kelslundberg** [comment]
Thanks, for adding this!

@johjacb - did you want me to spend some time looking at some of the layout options for the theme and thinking about how to break up the checklist into something more user friendly? Or was there something you saw during our meeting that you already wanted to use?

**Comment by e-jameson** [comment]
@c-schwarze how are we expecting these notifications to work? Do we load them every time from Banner or something?

**Comment by c-schwarze** [comment]
Yeah along those lines. We just need to create guidelines for what determines a notification and build that in our banner proc calls. At least that was my thought and I felt as though it was decently reasonable.

**Comment by johjacb** [comment]
It seems to me that we may need a little bit more discussion on this topic. I personally could use some clarification. And it looks like the notifications need some further discussion as well. Maybe we can put this on our agenda for the next meeting?

**Comment by kelslundberg** [comment]
Also, I'm meeting with @hamtim today to look at financial aid requirements so if we have a few minutes we can spend sometime with the checklist/notifications as well.

**Comment by kelslundberg** [comment]
Here's what we decided today:

Enrollment checklist will be it's own tab 
We're grouping list items by topic in separate sections or channels
Each section/channel will have some sort of progress bar or indicator of completion stations for that section

**Comment by kelslundberg** [comment]
@c-schwarze @johjacb Just checking in to see if we've made any progress on this since last meeting!

**Comment by johjacb** [comment]
yes, albeit minimal

**Comment by c-schwarze** [comment]
I added some fairly basic notification icons to the menus.

**Comment by kelslundberg** [comment]
@c-schwarze and @johjacb

Here's an idea for some content for the homepage widget that gets people to the checklist.

## **My Checklist**
View your enrollment checklist to get ready for your next term at Bethel.

**Todo Items** 
_Real time number here of how many todo items they have open_

I did add a channel (my-checklist) to the home tab with this content as kind of a place to start...

Maybe it could be styled to look something like these widgets?
![screen shot 2017-05-24 at 1 58 44 pm](https://cloud.githubusercontent.com/assets/6525391/26420875/a0f77242-4089-11e7-9bfb-00632b4e3ffb.png)

**Comment by kelslundberg** [comment]
Also, not sure about what would be the link..are these whole widgets designed to link somewhere?

**Comment by kelslundberg** [comment]
Too add a bit to this - we confirmed last week having a widget like this in the top section of the home tab.

There was talk of maybe adding the progress bar to it if that was possible so you could see how far along you were.

**Comment by kelslundberg** [comment]
Changing this to just "Checklist" since we're working out the dashboards in separate issues
